SkateShop.Maui/Entities/Cart/CartItemAddedMessage.cs

```diff
@@ -1,19 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace SkateShop.Maui.Entities.Cart
+﻿namespace SkateShop.Maui.Entities.Cart
 {
-    public class CartItemAddedMessage
-    {
-        public CartItem Item { get; }
-
-        public CartItemAddedMessage(CartItem item)
-        {
-            Item = item;
-        }
-    }
+    public record CartItemAddedMessage(CartItem Item);
```

Comment:
Thanks for adding this - someone asked about this during stream and I forgot to try it 😞